src/main/java/com/azure/cosmos/kafka/connect/sink/StructToJsonMap.java

```diff
@@ -73,11 +73,48 @@ public static Map<String, Object> toJsonMap(Struct struct) {
                 case STRUCT:
                     jsonMap.put(fieldName, toJsonMap(struct.getStruct(fieldName)));
                     break;
+                case MAP:
```

Comment:
Any other possible types that needs to be covered?

Reply:
Not really,  I think that should cover the cases.